doc/source/upstream-bug-fixes.rst

```diff
@@ -1,637 +1,139 @@
-|:Fix Released: :rn:`5.7.21-21`                                                                               |
-|:Upstream fix: N/A                                                                                           |
+|:Upstream Bug: :mysqlbug:`89840` - 60-80k connections causing empty reply for select                         |
+|:JIRA bug: :psbug:`314`                                                                                      |
+|:Upstream State: Verified (checked on 2018-11-20)                                                            |
+|:Fix Released: 8.0.12-2rc1                                                                                   |
+|:Upstream Fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
-|:Upstream bug: :mysqlbug:`90238` - Comparison of uninitailized memory in log_in_use                          |
-|:JIRA bug: :psbug:`3925`                                                                                     |
-|:Upstream state: No Feedback (checked on 2018-05-24)                                                         |
-|:Fix Released: :rn:`5.7.21-21`                                                                               |
-|:Upstream fix: N/A                                                                                           |
+|:Upstream Bug: :mysqlbug:`89607` - MySQL crash in debug, PFS thread not handling singals.                    |
+|:JIRA bug: :psbug:`311`                                                                                      |
+|:Upstream State: Verified (checked on 2018-11-20)                                                            |
+|:Fix Released: 8.0.12-2rc1                                                                                   |
+|:Upstream Fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
-|:Upstream bug: :mysqlbug:`89916` - hp_test1/hp_test2 are not built unless WITH_EMBEDDED_SERVER is defined    |
-|:JIRA bug: :psbug:`3845`                                                                                     |
-|:Upstream state: Verified (checked on 2018-05-24)                                                            |
-|:Fix Released: :rn:`5.7.21-21`                                                                               |
-|:Upstream fix: N/A                                                                                           |
-+-------------------------------------------------------------------------------------------------------------+
-|:Upstream bug: :mysqlbug:`89822` - InnoDB retries open on EINTR error only if innodb_use_native_aio is ...   |
```

Comment:
Resurrect

Reply:
It seems that "resurrect" comments in this file have not been addressed. OK to push the PR and address them later, but they must be addressed.